pytab/core.py

```diff
@@ -26,6 +26,7 @@
 from requests import get as reqGet
 
 from pytab import api, hwi, worker
+import yaml
```

Comment:
I would argue that importing yaml just to print detected hardware is a bit... overkill(?)